extension/vsc-js/test/suite/extension.test.js

```diff
@@ -0,0 +1,15 @@
+const assert = require('assert');
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+const vscode = require('vscode');
+// const myExtension = require('../extension');
+
+suite('Extension Test Suite', () => {
+	vscode.window.showInformationMessage('Start all tests.');
```

Comment:
perhaps we could remove the tests for now?

Reply:
these are auto-generated test files, should I still remove them?

Reply:
Perhaps there is a basic test we can perform? If that's difficult, then I would remove altogether, or keep the file but add a comment that tests are not implemented yet.